app/page.tsx

```diff
@@ -7,7 +8,7 @@ import Link from 'next/link';
 
 export default function LandingPage() {
   return (
-    <>
+    <body className='grid justify-center h-screen  bg-teal mx-8 mt-10' >
```

Comment:
I think Mora removed some of the body tags yesterday. There should always only be one. If I'm not mistaken, now we'd have two? One in the root layout and this one? Could you double check that, if that's the case, could you just use a div?